codalab/worker/docker_utils.py

```diff
@@ -261,6 +265,38 @@ def get_container_stats(container):
     return stats
 
 
+@wrap_exception('Unable to check Docker API for container')
+def get_container_stats_on_mac(container: docker.models.containers.Container):
```

Comment:
Why `_mac`?  Seems very specific.

Reply:
The existing method to retrieve container stats doesn't work on mac, so I created this method to develop on my machine. I'm experimenting to see if I can get the values from the existing method.